src/tests.rs

```diff
@@ -825,20 +825,6 @@ fn requesting_paid_file_with_no_lnd_returns_internal_error() {
   );
 }
 
-#[test]
-fn errors_contain_backtraces() {
-  let stderr = test(|context| async move {
-    context.write(".hidden", "");
-    let status = reqwest::get(context.files_url().join(".hidden").unwrap())
-      .await
-      .unwrap()
-      .status();
-    assert_eq!(status, StatusCode::NOT_FOUND);
-  });
-
-  assert_contains(&stderr, "agora::files::Files::check_path");
-}
-
```

Comment:
This tests backtrace printing. And that doesn't work as a unit test anymore, since backtraces are disabled. So I moved this test into `tests/smoke.rs`, which tests the `agora` executable compiled in non-test mode.